### PR TITLE
Document providing GCP & AWS access

### PR DESCRIPTION
**Pull request by yuvipanda**
Fixes https://github.com/2i2c-org/team-compass/issues/133

### CODE REVIEW

File: reference/cloud-access.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+We manage many projects across multiple cloud providers. This
+document defines our access policy, and is the canonical location
+for the projects we *do* have access to.
+
+## Access policy
+
+Every 2i2c engineer should have equal access to every cloud project
+we maintain. This prevents particular individuals from becoming
+single points of failure.
```
Comment (damianavila) [inline reference/cloud-access.md:11]: Super important to have a good bus factor!

File: reference/cloud-access.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+access. We should try to find ways around this, but if not,
+just do the paperwork.
+
+## Google Cloud Access
+
+On Google Cloud, we have a [2i2c organization](https://console.cloud.google.com/projectselector2/home/dashboard?organizationId=184174754493&supportedpurview=project),
+that contains some projects we are fully responsible for. Access
+to *all* projects in this organization can be granted by
+adding the user to the group `gcp-organization-admins@2i2c.org`
+in the [Google Workspace Admin dashboard](https://admin.google.com/ac/users).
```
Comment (damianavila) [inline reference/cloud-access.md:23]: When I try to access that link, it shows me the Choose Account page but selecting the 2i2c.org email is not going forward, any thoughts?

File: reference/cloud-access.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Note that this option is only available for engineers with a `@2i2c.org`
+account.
+
+For all other projects, we will need to make a manual entry in
+the project's [IAM Page](https://console.cloud.google.com/iam-admin/iam)
+for the engineer's `@2i2c.org` account, with `Owner` permissions.
+
+The canonical list of GCP projects we have access to is maintained
+[in this google sheet](https://docs.google.com/spreadsheets/d/1NSaAKLG2_njXxs6JlGUAhSWeHONz9QSGLVwEK790IZo/edit#gid=846555027)
```
Comment (damianavila) [inline reference/cloud-access.md:32]: The check links on that sheet are redirecting me to cloudbank or our 2i2c org stuff.
I can see the pangeo stuff looking around, but I can not see anything about meom.

File: reference/cloud-access.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## AWS Access
+
+Each 2i2c engineer should get an [IAM User Account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html)
+in every AWS account we have access to. This should have the broadest
+set of permissions as well, and the same permissions for everyone.
+
+The canonical list of AWS accounts we have access to is maintained
+[in this google sheet](https://docs.google.com/spreadsheets/d/1NSaAKLG2_njXxs6JlGUAhSWeHONz9QSGLVwEK790IZo/edit#gid=537065664)
```
Comment (damianavila) [inline reference/cloud-access.md:41]: Edited the title of that sheet to include an AWS reference.

File: reference/cloud-access.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+for the engineer's `@2i2c.org` account, with `Owner` permissions.
+
+The canonical list of GCP projects we have access to is maintained
+[in this google sheet](https://docs.google.com/spreadsheets/d/1NSaAKLG2_njXxs6JlGUAhSWeHONz9QSGLVwEK790IZo/edit#gid=846555027)
+
+## AWS Access
+
+Each 2i2c engineer should get an [IAM User Account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html)
+in every AWS account we have access to. This should have the broadest
+set of permissions as well, and the same permissions for everyone.
```
Comment (damianavila) [inline reference/cloud-access.md:38]: Do we want to have multiple users or just a generic 2i2c user?
Related: https://github.com/2i2c-org/pilot-hubs/issues/381

File: reference/cloud-access.md (hunk 3 above)
Comment (yuvipanda) [reply reference/cloud-access.md:32]: I invited you just now! Glad we caught this

File: reference/cloud-access.md (hunk 2 above)
Comment (yuvipanda) [reply reference/cloud-access.md]: Only 2i2c.org workspace admins can access this - I've clarified that.

File: reference/cloud-access.md (hunk 4 above)
Comment (yuvipanda) [reply reference/cloud-access.md:42]: yay, ty

File: reference/cloud-access.md (hunk 5 above)
Comment (yuvipanda) [reply reference/cloud-access.md]: I think each user should be unique. https://github.com/2i2c-org/pilot-hubs/issues/381 should be a 'service account' of sorts, I think.

**Comment by yuvipanda** [comment]
Thanks @damianavila, I've amended the docs based on your feedback!

**Review by sgibson91 — APPROVED** [review]
💃🏻 I love all these docs springing up

File: reference/cloud-access.md (hunk 2 above)
Comment (damianavila) [reply reference/cloud-access.md]: > I've clarified that in the doc now.

Thanks!

File: reference/cloud-access.md (hunk 3 above)
Comment (damianavila) [reply reference/cloud-access.md:33]: No problem!! Thanks again!

File: reference/cloud-access.md (hunk 5 above)
Comment (damianavila) [reply reference/cloud-access.md]: That makes sense to me.

**Review by damianavila — APPROVED** [review]
LGTM now!

**Comment by yuvipanda** [comment]
Thanks, @damianavila and @sgibson91!

**Pull request merged**